### PR TITLE
Merging in URL File/Tests+Skeletons

### PR DESCRIPTION
**Pull request by Kinneyj47**
I completed unit tests for the URL file. These tests make sure that a get request returns the correct url and utilizes the correct template. In order to get these tests to pass, I created template skeletons for the account creation, course creation and login pages. I also created a views.py skeleton, which has classes for the login, course creation and account creation page. The get method just returns a render of the correct skeleton template. Please let me know what everyone thinks. Hope everyone had a good Thanksgiving!

### CODE REVIEW

File: TAScheduler/urlTests.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ #       r=self.client.post('/admin/', {"username":"blah", "password":"blah"})
+ #       print(r.url)
+ #       self.assertEqual(r.url, "/admin/")
+    def test_loginTemplates(self):
+        r = self.client.get("/login/")
+        self.assertEqual(r.templates[0].name, "login.html")
+
+    def test_courseCreationTemplates(self):
+        r = self.client.get("/coursecreation/")
+        self.assertEqual(r.templates[0].name, "coursecreation.html")
```
Comment (LLCrafter) [inline TAScheduler/urlTests.py]: I think we're gonna be calling this courseManagement.html as it'll need to cover more later

File: TAScheduler/urlTests.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    def test_accountCreationTemplates(self):
+        r = self.client.get("/accountcreation/")
+        self.assertEqual(r.templates[0].name,  "accountcreation.html")
```
Comment (LLCrafter) [inline TAScheduler/urlTests.py]: Same goes for this, but maybe userManagement.html

**Review by LLCrafter — COMMENTED** [review]
So just like in my comments, the pages aren't just going to be for creation, but they will need to cover management in general, so it might be a good idea to reflect that early on.

**Comment by LLCrafter** [comment]
I realize that, that may have been confusing given that the task talks about course/account creation.

**Comment by Kinneyj47** [comment]
Sounds good, I will update it.

**Comment by Kinneyj47** [comment]
Updated, thanks for the feedback!

**Comment by LLCrafter** [comment]
Awesome, I didn't spot any trouble, so I think its good.

**Comment by gatesjm-uwm** [comment]
:+1:, we may want to split out the views to separate files in the future, but for the moment this all looks good - nice work!

**Pull request merged**